server/clip_server/model/clip_onnx.py

```diff
@@ -17,8 +17,11 @@
 
 
 class CLIPOnnxModel:
-    def __init__(self, name: str = None):
-        if name in _MODELS:
+    def __init__(self, name: str = None, model_paths: str = None):
+        if model_paths is not None:
```

Comment:
The `name` param should always be one of the `_MODELS` even if the given `model_paths` is not empty.